js/modules/resolution.go

```diff
@@ -103,24 +104,17 @@ func (mr *ModuleResolver) resolveLoaded(basePWD *url.URL, arg string, data []byt
 	if cached, ok := mr.cache[specifier.String()]; ok {
 		return cached.mod, cached.err
 	}
-	prg, _, err := mr.compiler.Parse(string(data), specifier.String(), false)
+	prg, _, err := mr.compiler.Parse(string(data), specifier.String(), false, true)
```

Comment:
Shouldn't the last `true` be computed?

```suggestion
	prg, _, err := mr.compiler.Parse(string(data), specifier.String(), false, isESM(prg))
```

Reply:
:facepalm: